docs/source/linchpincli_linchpin_validate.rst

```diff
@@ -1,27 +0,0 @@
-Validate
```

Comment:
This functionality is temporarily disabled in the current rework. Let's comment this out as the functionality will return in the near future.